### PR TITLE
RD-6496 Fix system tests failures related to theming changes

### PR DESCRIPTION
**Pull request by Vorbert-Kruk**
## Description

Changes consists of:
- Adjusting menu item icon name within `user_menu_spec` test file
- Executing `clickButton` function within narrowed context

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

1. System tests for adjusted test scenarios
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3967/)
(they are failing not because of the theming changes)

2. System tests for new failures, that appeared
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3972/)

## Documentation

N/A

### CODE REVIEW

File: test/cypress/integration/widgets/sites_spec.ts [hunk 1]
```diff
@@ -142,14 +143,16 @@ describe('Sites Management', () => {
         // use map to specify location
         cy.get('form :nth-child(2) > .field > .ui > button').click();
         cy.get('.leaflet-container').click();
-        const location = '0.013732910024768903, -0.8789062500000001';
+        const location = '0.05218505137865288, -0.8789062500000001';
```
Comment (Vorbert-Kruk) [inline test/cypress/integration/widgets/sites_spec.ts:146]: ### Note
This issue was not related to the theming changes, but as it appeared, I've fixed it at once 💪

File: test/cypress/integration/widgets/sites_spec.ts [hunk 2]
```diff
@@ -187,7 +190,7 @@ describe('Sites Management', () => {
     });
 
     it('update a site with location changed with map', () => {
-        const location = '32.11282228909443, 33.92578125000001';
+        const location = '32.145385562629315, 33.92578125000001';
```
Comment (Vorbert-Kruk) [inline test/cypress/integration/widgets/sites_spec.ts:193]: ### Note
This issue was not related to the theming changes, but as it appeared, I've fixed it at once 💪

File: test/cypress/integration/widgets/user_groups_spec.ts [hunk 3]
```diff
@@ -52,8 +55,10 @@ describe('User group management widget', () => {
         cy.contains('tr', groupName).contains('.label.blue', '1');
 
         cy.log('Verifying group users and tenants can be removed');
-        cy.contains('tr', groupName).click();
-        cy.get('.remove').click({ multiple: true });
+        cy.contains('tr', groupName).within(() => {
+            cy.get('td').click();
+            cy.get('.remove').click({ multiple: true });
+        });
```
Comment (Vorbert-Kruk) [inline test/cypress/integration/widgets/user_groups_spec.ts:61]: ## Note

This issue is was also not related to the theming changes.
It seemed that by clicking `tr` element, user group section wasn't expanding - but it appeared that when clicking the inner `td` element, issue doesn't occur

Maybe there were some changes, which have implemented addition `event.stopPropagation()` for the table component 🤔 

Either way, I've taken care about this issue 🎉

File: test/cypress/integration/widgets/sites_spec.ts (hunk 1 above)
Comment (qooban) [reply test/cypress/integration/widgets/sites_spec.ts:146]: I'm always curious about this location instability 🙂 It hits us from time to time. Where it comes from? Browser (Electron) changes maybe? 🤔

**Review by qooban — APPROVED** [review]

File: test/cypress/integration/widgets/user_groups_spec.ts (hunk 3 above)
Comment (qooban) [reply test/cypress/integration/widgets/user_groups_spec.ts]: @PatrycjaKubiczek Maybe something related to #2277 ? 🤔

File: test/cypress/integration/widgets/sites_spec.ts (hunk 1 above)
Comment (Vorbert-Kruk) [reply test/cypress/integration/widgets/sites_spec.ts:146]: I'm also curious about that 🤔 
Last time I've tried to deeply investigate it, but still didn't find the cause of it 

If I'd guess, I'd say that the browser version or the viewport size may be interfering here 🤔

**Pull request merged**